MatrixSDK/VoIP/CallKit/MXCallKitAdapter.m

```diff
@@ -0,0 +1,285 @@
+/*
+ Copyright 2017 Vector Creations Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MXCallKitAdapter.h"
+
+@import AVFoundation;
+@import CallKit;
+@import UIKit;
```

Comment:
We will need to use ```#if TARGET_OS_IPHONE``` compilation flag as the sdk can be also used for macos dev.
You can quickly check mac os build thanks to the validation the sdk pod. Run ```pod lib lint MatrixSDK.podspec``` in the sdk folder.